lib/Models/Subscriber.php

```diff
@@ -104,6 +105,14 @@ function isWooCommerceUser() {
     return (bool)$this->is_woocommerce_user;
   }
 
+  function getLinkToken() {
```

Comment:
This is a good idea but it will require many changes and make this PR bigger, especially if we want to inject that service using DI. What about doing that in a separate task?

Reply:
If this weren't a high priority task, I would argue we should do the refactor now. We should always make the code better. [Do you know the boy scout rule?](https://medium.com/@biratkirat/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385) However this is task has high priority, so we probably don't want to wait for the refactor. But on the other hand, we've been waiting for it a couple of weeks already so it can wait a bit longer.

Anyway, your call. You decide.

Reply:
I will create a separate task and work on it right away. This way we can ship this fix ASAP.